compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -721,6 +721,15 @@ fn link_dwarf_object<'a>(
     }
 }
 
+#[derive(Diagnostic)]
+#[diag(codegen_ssa_linker_output)]
+/// Translating this is kind of useless. We don't pass translation flags to the linker, so we'd just
+/// end up with inconsistent languages within the same diagnostic. Ideally we'd accept strings in
+/// `emit_warning`, but I didn't feel like doing that much of a refactor.
```

Comment:
No, the fact that doing this is annoying is good, if we accepted strings there we'd just be back at untranslatable diagnostics. Having to do this explicit hack for something I expect to show up very rarely seems like a good thing, so I'd prefer removing the last sentence :)